src/motion/rt_propagation.F

```diff
@@ -463,7 +463,7 @@ SUBROUTINE rt_initial_guess(qs_env, force_env, rtp_control)
          ALLOCATE (qs_env%rtp)
          CALL get_qs_env(qs_env, matrix_s=matrix_s)
          CALL rt_prop_create(qs_env%rtp, qs_env%mos, qs_env%mpools, dft_control, matrix_s(1)%matrix, &
-                             rtp_control%linear_scaling, qs_env%admm_env%mos_aux_fit)
+                             rtp_control%linear_scaling)
```

Comment:
I think you have to add a branch here, one with ADMM, one without ADMM because made mos_aux became an optional parameter

Reply:
@fstein93 Thanks for the hint.